### ISSUE TITLE
Larger LLM for people with a lot of RAM?

### ISSUE THREAD

**Issue by kaxlabs** [ohmplatform/FreedomGPT#33]
Is there any way we can download a larger LLM if we have enough RAM and resources to support it? I currently have 128GB for reference.

**Comment by ohmguru** [comment]
Yes, please feel free to submit a PR. It would be relatively easy to give people an option to use the larger LLM if they have a more powerful machine.

**Comment by ItsPi3141** [comment]
I support this issue! I need to be able to use Alpaca 30B and Alpaca Native 13B on my computer!!
